doc/src/tutorial/gotchas.rst

```diff
@@ -239,6 +239,8 @@ object and a Python object, you get a SymPy object, but whenever you combine
 two Python objects, SymPy never comes into play, and so you get a Python
 object.
 
+.. warning:: SymPy Live wraps both 1s in the expression ``1 + 1`` by Integer, so the example doesn't show what it is supposed to show. Similar issue occurs in few following examples as well. This occurs doesn't occur in actual desktop environment.
```

Comment:
This should be more general. Maybe a section on SymPy live explaining that expressions are sympyfied automatically.